roottest/root/io/tuple/CMakeLists.txt

```diff
@@ -1,7 +1,6 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-ROOTTEST_ADD_OLDTEST()
+if(NOT MSVC)
```

Comment:
As far as I can tell the test was run on MSVC.  It is meant to check that all the files are readable and all the platforms.

Reply:
According to @bellenot - none of Makefile-based tests were running on Windows.
And this test failing now.
Therefore disable it and Bertrand may try to debug it later.

Reply:
Let's try this one :)

Reply:
I.e. you are right the Makefile test were not running because there is no 'gmake' on Windows (and thus not necessarily because they don't work).

Reply:
I had not noticed you already tried.  What was the error?

Reply:
I create PR https://github.com/root-project/root/pull/18815 to reproduce problem.
The output is:
```
Writing tuple.root
  Reading tuple.root
  Error in <read>: Component 0 incorrect read 0 instead of 33
  Error in <read>: Component 1 incorrect read 0.000000 instead of 66.666667
  Reading tuple.macos.root
  Error in <read>: Component 0 incorrect read 0 instead of 33
  Error in <read>: Component 1 incorrect read 0.000000 instead of 66.666667
  Error in <read>: Component 2 incorrect read -85 instead of 13
  Reading tuple.x86_64.root
  Error in <read>: Component 0 incorrect read 0 instead of 33
  Error in <read>: Component 1 incorrect read 0.000000 instead of 66.666667
  Error in <read>: Component 2 incorrect read -85 instead of 13
```

Reply:
That's sad but clear :(